pkg/kubelet/kuberuntime/kuberuntime_container.go

```diff
@@ -714,15 +722,16 @@ func (m *kubeGenericRuntimeManager) killContainer(pod *v1.Pod, containerID kubec
 		"containerName", containerName, "containerID", containerID.String(), "gracePeriod", gracePeriod)
 
 	err := m.runtimeService.StopContainer(containerID.ID, gracePeriod)
-	if err != nil {
+	if err != nil && !crierror.IsNotFound(err) {
 		klog.ErrorS(err, "Container termination failed with gracePeriod", "pod", klog.KObj(pod), "podUID", pod.UID,
 			"containerName", containerName, "containerID", containerID.String(), "gracePeriod", gracePeriod)
+		return err
 	} else {
```

Comment:
You should remove this else.

Reply:
To clarify, the else on line 729 needs removed but to keep the log line.

Reply:
Done